src/helpers/terminal_helpers.sh

```diff
@@ -157,7 +157,7 @@ function prompt_yn {
 
 
 # Run a command, prints command and output
-function run_command {
+function run_git_command {
   local cmd="$*"
   print_command "$cmd"
```

Comment:
I think `print_command` should be renamed to `print_git_command` since it prepends the git branch.